src/main/java/uk/ac/cam/cl/dtg/segue/comm/EmailManager.java

```diff
@@ -273,6 +273,75 @@ public void sendCustomEmail(final RegisteredUserDTO sendingUser, final String co
         log.info(String.format("Admin user (%s) added %d emails to the queue. %d were filtered.", sendingUser.getEmail(),
                 allSelectedUsers.size() - numberOfFilteredUsers, numberOfFilteredUsers));
     }
+
+
+    /**
+     * @param sendingUser
+     * 				- the user object for the user sending the email
+     * @param plaintextTemplate
+     * 				- the plainText of the email
+     * @param htmlTemplate
+     *              - the html of the email
+     * @param allSelectedUsers
+     * 				- the users to send email to
+     * @param emailSubject
+     *              - the subject of the email
+     * @param emailType
+     * 				- the type of email to send (affects who receives it)
+     * @throws SegueDatabaseException
+     * 				- a segue database exception
+     * @throws ContentManagerException
+     * 				- a content management exception
+     */
+    public void sendCustomContentEmail(final RegisteredUserDTO sendingUser, final String plaintextTemplate, final String htmlTemplate,
+                                final String emailSubject, final String overrideFromAddress, final List<RegisteredUserDTO> allSelectedUsers, final EmailType emailType) throws SegueDatabaseException,
+            ContentManagerException {
+        Validate.notNull(allSelectedUsers);
+        Validate.notNull(plaintextTemplate);
+        Validate.notNull(htmlTemplate);
+        Validate.notNull(emailSubject);
+
+        EmailTemplateDTO emailContent = new EmailTemplateDTO();
```

Comment:
Is there a reason we don't just use the DTO provided by the client directly? This is a staff-only endpoint, and so we do trust the value I think? That makes it more future-proof and would save having to pass all these properties in one-by-one; what happens if we need to change the From address and the From Name too?